modules/installation-user-infra-machines-iso.adoc

```diff
@@ -82,6 +83,16 @@ For example, setting
 * If your system has multiple network interfaces, you can add multiple IP address configurations by specifying multiple `ip=` entries.
 * If your system has multiple network interfaces, you can use a combination of DHCP and static IP configurations. You must specify the interface to use, for example `ip=enp1s0:dhcp ip=10.10.10.2::10.10.10.254:255.255.255.0:core0.example.com:enp2s0:none`.
 * You can provide multiple DNS servers by adding a `nameserver=` entry for each server, such as `nameserver=1.1.1.1 nameserver=8.8.8.8`.
+<6> Optionally, you can bond multiple network interfaces to a single interface
+using the `bond=` option, as described here:
+* The syntax is: `bond=name[:slaves][:options]`.
+* In the following example, _name_ is the bonding device name (bond0), _slaves_
+represents a comma-separated list of physical (ethernet) interfaces (`em1,em2`),
+and _options_ is a comma-separated list of bonding options.
+* Note that once you
+create a bonded interface using `bond=`, you must specify how the IP address
```

Comment:
Should this sentence work in that you specify how the IP is assigned using the `ip=` option?

Reply:
Sure. I'll add "...using the ip= option"

Reply:
Done.